scripts/create-test-data.js

```diff
@@ -112,6 +103,10 @@ module.exports = function() {
     buildStart() {
       this.addWatchFile(segmentsDir);
       this.addWatchFile(manifestsDir);
+
+      [].concat(getSegments())
```

Comment:
With the newer version of rollup that we use we have to watch all the files in the segment/manifests dir. Without this change we only watch files that are added to the segment/manifest directory for changes. Said another way, we only watch new files for changes, while changes to existing files will not trigger a rebuild. This change allows existing file changes to trigger a rebuild.